extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -188,8 +188,8 @@ export class DoubleclickA4aEligibility {
     if (isExperimentOn(win, dfDepRollbackExperiment) ||
         experimentId ==
         DOUBLECLICK_EXPERIMENT_FEATURE.DF_DEP_HOLDBACK_EXPERIMENT) {
-      return !hasUSDRD &&
-          (!useRemoteHtml || !!element.getAttribute('rtc-config'));
+      return !!element.getAttribute('rtc-config') ||
```

Comment:
In the event RTC and uSDRUD are both present, can we warn that uSDRUD is being ignored?

Reply:
Given we warn earlier, this seems fine